docs/teams/independent.html

```diff
@@ -29,6 +29,16 @@
                 </div>
             </div>
         </div>
+        <div id="hive-mind" class="collapsible">
+            <button class="header" style="background-color:var(--independent);"><i class="icon-hive-mind"></i>Hive Mind</button>
```

Comment:
The header should have the monster colour too
```suggestion
            <button class="header" style="background-image:linear-gradient(110deg, var(--independent) 0%, var(--independent) 70%, var(--monster) 70%, var(--monster) 100%);"><i class="icon-hive-mind"></i>Hive Mind</button>
```